dev-java/protobuf-java/protobuf-java-3.21.5.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 2008-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Tests not enabled, depend on com.google.truth which is not packaged
+# https://github.com/protocolbuffers/protobuf/blob/v21.4/java/core/pom.xml#L35-L40
+JAVA_PKG_IUSE="doc source"
+MAVEN_ID="com.google.protobuf:${PN}:${PV}"
```

Comment:
Does java-ebuilder understand that syntax?

Reply:
No idea yet while pretty confident it will work.
It needs more testing before I could consider it as valuable.

Reply:
Updated below, we consider it okay for now.